build/commands/lib/config.js

```diff
@@ -287,9 +287,15 @@ Config.prototype.buildArgs = function () {
     args.last_chrome_installer = this.last_chrome_installer
   }
 
+  if (process.platform === 'darwin') {
+    if (this.use_goma && this.gomaServerHost) {
```

Comment:
@mihaiplesa we should probably provide an error to the user if `use_goma` is set without `gomaServerHost` because it's very confusing when it tries to connect to chromium

Reply:
this is the problem I ran into when I initially tried to connect and then it overwrote my creds with the wrong values